lib/index.ts

```diff
@@ -165,8 +165,8 @@ async function getManifestFiles(
       return {
         name: path.basename(g),
         path: path.dirname(g),
-        contents: Buffer.from(contents[i].stdout).toString("base64"),
+        contents: Buffer.from(contents[i].stdout),
       };
     })
-    .filter((i) => i.contents !== "");
+    .filter((i) => i.contents.toString() !== "");
```

Comment:
is there a more Buffer-y way to do it? for example `i.contents.length > 0`

Reply:
Not sure what a "Buffer-y way" is. For this case both constructs are fine. I have no preference either way. I can change it.

Reply:
I prefer to not call toString if not necessary, if it's a Buffer then let's use it as a Buffer :)